### PR TITLE
Attendance Sebastian 2021-09-03

### PR DESCRIPTION
**Pull request by elrocetas**

### CODE REVIEW

File: SebastianHerrera/HackerrankSpring/src/main/java/com/globant/hackerrankspring/exception/CustomExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * @created 06/09/21
+ */
+
+@SuppressWarnings({"unchecked", "rawtypes"})
+@ControllerAdvice
+public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
+
+    @ExceptionHandler(Exception.class)
+    public final ResponseEntity<Object> handleAllExceptions(Exception ex, WebRequest request) {
+        List<String> details = new ArrayList<>();
```
Comment (luisalanmz) [inline SebastianHerrera/HackerrankSpring/src/main/java/com/globant/hackerrankspring/exception/CustomExceptionHandler.java:27]: If you only add one thing into a list, I suggest to you use Collections.singletonList

The reason :
https://www.it-swarm-fr.com/fr/java/arrays.aslist-vs-collections.singletonlist/1049748648/

**Review by luisalanmz — COMMENTED** [review]
one suggestion

**Review by luisalanmz — APPROVED** [review]
LGTM!

File: SebastianHerrera/HackerrankSpring/src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+server:
+  port: 8080
+request:
+  api: /Hackerrank
+api:
+  url:
+    check-palindrome: /CheckPalindrome
```
Comment (luisod78) [inline SebastianHerrera/HackerrankSpring/src/main/resources/application.yml:7]: awesome!

File: SebastianHerrera/HackerrankSpring/src/main/java/com/globant/hackerrankspring/controller/HackerrankDay4.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    private CheckAnagramService checkAnagram;
+
+    public HackerrankDay4(PalindromeService palindromeValidate, CheckPatternService syntaxChecker,
+                          CheckAnagramService checkAnagram) {
+        this.palindromeValidate = palindromeValidate;
+        this.syntaxChecker = syntaxChecker;
+        this.checkAnagram = checkAnagram;
+    }
+
+    @PostMapping(value = "${api.url.check-palindrome}", produces = MediaType.APPLICATION_JSON_VALUE)
```
Comment (luisod78) [inline SebastianHerrera/HackerrankSpring/src/main/java/com/globant/hackerrankspring/controller/HackerrankDay4.java:44]: I'm curious... all your 3 methods are getSomthing... but all 3 are PostMapping??? why?? r u creating something?

**Review by luisod78 — APPROVED** [review]

**Pull request merged**